qiskit/circuit/quantumcircuit.py

```diff
@@ -604,6 +604,7 @@ def draw(self, scale=0.7, filename=None, style=None, output=None,
             idle_wires (bool): Include idle wires. Default is True.
             with_layout (bool): Include layout information, with labels on the physical
                 layout. Default is True.
+            fold (int): In MPL is the amount of operations before folding. Default is 25.
```

Comment:
```suggestion
            fold (int): In 'mpl' and 'text' is the number of layers before folding. Default is 25.
```
is the default the same for both mpl and text?

Reply:
I updated the docstring in bbe08ac8